sockeye/data_io.py

```diff
@@ -1022,6 +1029,22 @@ def ids2strids(ids: Iterable[int]) -> str:
     return " ".join(map(str, ids))
 
 
+def ids2tokens(token_ids: Iterable[int],
+               vocab_inv: Dict[int, str],
+               exclude_set: Set[int] = set()) -> List[str]:
```

Comment:
The typical use case of this function is to join the resulting list on C.TOKEN_SEPARATOR. In that case its better to return a generator (`return (tok for token_id ....)`).

Reply:
I did not know you could create a generator in this way!